srcpkgs/xdg-desktop-portal/template

```diff
@@ -1,19 +1,19 @@
 # Template file for 'xdg-desktop-portal'
 pkgname=xdg-desktop-portal
-version=1.8.1
+version=1.14.1
```

Comment:
can we also split xdg-desktop* updates to a separate PR? Do they still work with GNOME 41?

Reply:
xdg-desktop-portal will work with gnome 41
xdg-desktop-gtk will not work properly with gnome 41 without xdg-desktop-gnome.

Reply:
Then let's leave it in this PR